### PR TITLE
Added the 4.2.6 version and formatted the release notes 

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Added the 4.2.6 version and formatted the release notes according to the new format.

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [X] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

**Review by drothery-edb — DISMISSED** [review]
Look over my changes and if they look ok to you, go ahead and merge.

File: product_docs/docs/pgpool/4.2/pgpool_rel_notes/index.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+| ------------------------- | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | 
+| [4.2.6](09_426_rel_notes) | 2021 Dec 01  | Upstream [4.2.6](https://www.pgpool.net/docs/42/en/html/release-4-2-6.html#bug-fixes)                                                                                                                                     |
+| [4.2.5](10_425_rel_notes) | 2021 Jun 10  | Upstream [4.2.5](https://www.pgpool.net/docs/42/en/html/release-4-2-5.html) and [4.2.4](https://www.pgpool.net/docs/42/en/html/release-4-2-4.html)                                                                                  |
+
+Often only select issues are included in the upstream merges. The specific issues included in the merges are listed in the release note topics.
```
Comment (drothery-edb) [inline product_docs/docs/pgpool/4.2/pgpool_rel_notes/index.mdx:16]: Is this true? I know it was for PEM, but is it also true for pgpool?

**Comment by nidhibhammar** [comment]
> 

The changes look good to me. The only question I have is we are providing the link to upstream merge release notes in the index.mdx page and also in the specific version release notes page. Do we need it at both places?

File: product_docs/docs/pgpool/4.2/pgpool_rel_notes/index.mdx (hunk 1 above)
Comment (nidhibhammar) [reply product_docs/docs/pgpool/4.2/pgpool_rel_notes/index.mdx]: removed this line as per our dicussion

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**